core/controllers/reader.py

```diff
@@ -615,11 +624,12 @@ class LearnerIncompleteActivityHandler(base.BaseHandler):
 
     @acl_decorators.can_access_learner_dashboard
     def delete(self, activity_type, activity_id):
```

Comment:
Just for me to know, when is this handler called in the frontend?

Reply:
When the user deletes a topic in the Current Goals section, this handler will be called.

Reply:
This handler is used when the learner wants to remove any story/topic from the incomplete list -- in the Topic in Progress tab.